sodiumpp/include/sodiumpp/locked_string.h

```diff
@@ -22,13 +22,13 @@ class locked_string final
              * for move construction for std::string use static methods:
              * move_from_locked_string() and move_from_not_locked_string
              */
-            explicit locked_string(std::string && str) noexcept; ///< only move, not call mlock()
+            //explicit locked_string(std::string && str) noexcept; ///< only move, not call mlock()
     public:
             locked_string() = default;
```

Comment:
We could remove that, because
- no reason to create zero-size locked strings, it's a mistake (yeap now secret_key class does that but it's bug we need to fix then)
- and it would require a bit other destructor logic to not "unlock" not locked memory